templates/homepage.html

```diff
@@ -3,8 +3,6 @@
 {% load cache %}
 {% load i18n %}
 
-{% block head_title %}Reusable apps, sites and tools directory{% endblock %}
```

Comment:
Why did this get removed? It doesn't look like the new code renders this information when hovering now

Reply:
> Why did this get removed? 

The `page_metadata()` template tag adds this title by default if the page does not have an explicit title.
See:
https://github.com/djangopackages/djangopackages/blob/ebc2dd4f7521199f98208e1ccf1ab2bde396bb6d/core/templatetags/page_metadata_tags.py#L13

partial reason: https://github.com/djangopackages/djangopackages/pull/869#discussion_r1032785561

> It doesn't look like the new code renders this information when hovering now

Do you mean hovering over the browser tab? or something else?

Reply:
@saadmk11  On the browser tab it looked like it’s missing.

Reply:
I am able to see it.
![Screenshot from 2022-11-26 20-37-15](https://user-images.githubusercontent.com/24854406/204094284-7c908bce-9d05-4ee0-a488-5a6ea12e2a02.png)

And the html source looks like this for the home page:

```html
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="Django Packages stores information on fetched packages and provides easy comparison tools for them. Public APIs include PyPI, Github, and BitBucket." />
<meta name="keywords" content="Django, Django Apps, Packages, Tools, Django Sites, Django Resources" />
<title>Django Packages : Reusable apps, sites and tools directory for Django</title>
<meta property="og:type" content="website" />
<meta property="og:site_name" content="Django Packages" />
<meta property="og:title" content="Django Packages : Reusable apps, sites and tools directory for Django" />
<meta property="og:description" content="Django Packages stores information on fetched packages and provides easy comparison tools for them. Public APIs include PyPI, Github, and BitBucket." />
<meta property="og:url" content="http://127.0.0.1:8000/" />
<meta property="og:image" content="http://127.0.0.1:8000/static/img/logo_squares.png" />
<meta property="og:image:alt" content="Django Packages logo" />
<meta property="og:image:width" content="195" />
<meta property="og:image:height" content="93" />
<meta property="og:image:type" content="image/png" />
<meta name="twitter:card" content="summary" />
```

Reply:
Sorry, it looks like I was incorrect / missed something when I was checking it out. I'm seeing it now 🤷‍♂️ 

Change looks good!